src/_artifacts.tf

```diff
@@ -0,0 +1,18 @@
+resource "massdriver_artifact" "fhir" {
+  field                = "azure_fhir_service"
+  provider_resource_id = azurerm_healthcare_fhir_service.main.id
+  name                 = "Azure FHIR Service ${var.md_metadata.name_prefix} (${azurerm_healthcare_fhir_service.main.id})"
+  artifact = jsonencode(
+    {
+      data = {
+        infrastructure = {
+          ari = azurerm_healthcare_fhir_service.main.id
+        }
+        authentication = {
+          authority = azurerm_healthcare_fhir_service.main.authentication[0].authority
```

Comment:
ahh this must be the cert.

@mclacore if so, when does it expire, does it auto-rotate? If Azure doesn't rotate it for us and we'd have to run this bundle again to get a new cert, that's why we punted TLS in GCP.